### ISSUE TITLE
chats are "locked"

### ISSUE THREAD

**Issue by fsheidaiee** [UnigramDev/Unigram#2160]
in chats when switching to another contact or channel or group the chat display doesn't change and remains at the previous chat windows please fix it
(by the way it doesn't happen everytime but sometimes it happens)

**Comment by MarcAnt01** [comment]
@fsheidaiee can you specify the version?

**Comment by fsheidaiee** [comment]
> @fsheidaiee can you specify the version?

7.2.5840.0

**Comment by bassclefstudio** [comment]
I have noticed the same issue, to the extent where at some point a sent message will seem to 'lock up' the messages displayed so that the newly sent/received messages don't get added to the view (but you see the notifications and the message text on the chats pane on the left) and changing the chat does nothing. Only solution I've found is to restart the app.

Same version - 7.2.5840.0 x64, on Windows 10 20H2.
